src/base/const_string.h

```diff
@@ -16,6 +16,7 @@ namespace base {
 
 class Literal {
  public:
+  inline const char* c_str() const { return str_; }
```

Comment:
In `ReplaceTildeInPath` from file_cache.cc.
`GetHomeDir()` returns a `Literal` thats why I should use `c_str()` to create a `Path` form it.